python/src/GaussProductExperiment_doc.i.in

```diff
@@ -2,25 +2,87 @@
-marginalDegrees : sequence of positive int
-    Numbers of points :math:`N_i` for each direction. Then, the total number of
-    points generated is :math:`cardI=\prod_{i=1}^n N_i`. If not specified,
-    each value :math:`N_i` is :math:`5`.
+marginalSizes : sequence of positive int
+    Numbers of points :math:`s_j` for each direction. Then, the total number of
+    points generated is :math:`\textrm{card}(I)=\prod_{j = 1}^{n_x} s_j`. 
+    By default, the value of :math:`s_j` is equal to :math:`5`.
+    The default marginal size is defined in the 
+    `GaussProductExperiment-DefaultMarginalSize` key of the :class:`~openturns.ResourceMap`. 
 distribution :
-    Distribution :math:`\mu` of dimension :math:`n` with an independent copula.
+    Distribution :math:`\mu` of dimension :math:`n_x` with an independent copula.
 
 Notes
 -----
-The number of points for each direction doesn't change and the :meth:`generate`
-method always gives the same sample.
+The tensor product design of experiments (DOE) is based on a collection of marginal multidimensional elementary designs of experiments. It is anisotropic in the sense that each marginal DOE does not necessarily have the same size. Using more points in one component may be useful if we want to get more accurate results in that particular direction. 
+
+The Gauss tensor product experiment uses Gauss points in each direction. For each marginal, the algorithm computes the family of orthogonal polynomials depending on the marginal distribution using the :class:`~openturns.StandardDistributionPolynomialFactory` class. The input distribution must have an independent copula.  
+
+The :meth:`generate` method always returns the same sample, i.e. this is a 
+deterministic DOE.
+
+Let :math:`\mathcal{X} \subset \mathbb{R}^{n_x}` be the integration domain and let :math:`g : \mathcal{X} \rightarrow \mathbb{R}^{n_y}` be an integrable function. Let :math:`f : \mathcal{X} \rightarrow \mathbb{R}` be a probability density function. The Gauss tensor product experiment produces an approximation of the integral:
+
+.. math::
+
+    \int_{\mathcal{X}} g(\boldsymbol{x}) f(\boldsymbol{x}) d\boldsymbol{x} \approx \sum_{i = 1}^{s_t} w_i g\left(\boldsymbol{x}_i\right)
+
+where :math:`s_t \in \mathbb{N}` is the size of the tensor product design of experiments, :math:`w_1, ..., w_{s_t} \in \mathbb{R}` are the weights and :math:`\boldsymbol{x}_1, ..., \boldsymbol{x}_{s_t} \in \mathbb{R}^{n_x}` are the nodes. 
+
+Let :math:`\boldsymbol{k} = \left(k_1, ..., k_{n_x}\right) \in (\mathbb{N}^\star)^{n_x}`  be a multi-index where :math:`\mathbb{N}^\star = \{1, 2, ..., \}` is the set of natural numbers without zero. For each marginal index :math:`j = 1, ..., n_x`, let :math:`s_j \in \mathbb{N}` be its size, representing the number of nodes in the marginal quadrature. Let :math:`\left\{ Q_{s_j}^{(1)} \right\}_{j = 1, ..., n_x}` be the collection of marginal experiments where the exponent :math:`(1)` denotes univariate quadrature rule. 
```

Comment:
most of these lines are way too long and do not ease the review, please skip lines

Reply:
Ok. Fixed.